bindings/python/CMakeLists.txt

```diff
@@ -70,8 +70,10 @@ endforeach()
 
 # Generate stubs
 if(GENERATE_PYTHON_STUBS)
+  python_build_get_target(python_build_target_name)
   load_stubgen()
   generate_stubs(
```

Comment:
Add a comment to explain generate_stubs dependencies.

I copy/paste the comment from pinocchio as an example :
```cmake
    # Set PYWRAP and PROJECT_NAME as stubs dependencies.
    # PROJECT_NAME is mandatory (even if it's a PYWRAP dependency)
    # to find PROJECT_NAME name DLL on windows.
```

Reply:
Good idea